src/fal/cli/flow_runner.py

```diff
@@ -50,6 +50,7 @@ def run_threaded(
         OriginGraph,
         FilteredGraph,
         PlannedGraph,
+        ShuffledGraph,
```

Comment:
I'd go with `Reordered` or `Connected` instead of `Shuffled`, no strong opinions though (shuffling is generally used in the context of the arrangement being random).

Reply:
Agreed, I will change it!